sycl/test-e2e/Graph/Explicit/add_nodes_after_finalize.cpp

```diff
@@ -1,10 +1,13 @@
 // REQUIRES: level_zero, gpu
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
+// RUN: env ZE_DEBUG=4 %{run} %t.out
```

Comment:
Could of ways I think we could improve this

1) Only do `ZE_DEBUG=4` when the target is level zero, so we can use these tests for other targets in future - see [Basic/alloc_pinned_host_memory.cpp](https://github.com/intel/llvm/blob/sycl/sycl/test-e2e/Basic/alloc_pinned_host_memory.cpp#L4)

2) We should check the output of these run by piping them into [FileCheck](https://llvm.org/docs/CommandGuide/FileCheck.html), not just that they don't crash.  See [test-e2e/Plugin/level-zero-event-leak.cpp](https://github.com/intel/llvm/blob/sycl/sycl/test-e2e/Plugin/level-zero-event-leak.cpp#L110)

I think this would up with something like this, but I've not tried it.
```
// RUN: env ZE_DEBUG=4 %{run} %t.out %if ext_oneapi_level_zero %{2>&1 | FileCheck %s}
// CHECK-NOT: LEAK